lib/metasploit/framework/ssh/platform.rb

```diff
@@ -52,7 +52,7 @@ def self.get_platform_info(ssh_socket, timeout: 10)
                 info = info.map { |item| item.strip }
                 info = info.join(', ').to_s
                 # Windows
-              elsif info =~ /command not found|is not recognized as an internal or external command/
+              elsif info =~ /command not found|is not recognized as an internal or external command|is not recognized as the name of a cmdlet, function, script file, or operable program./
```

Comment:
I would suggest removing last word as for some version, the response is following: 
```
[+] 172.16.187.139:22 - Success: 'ms:kali' 'id : The term 'id' is not recognized as the name of a cmdlet, function, script file, or operable   program. Check the spelling of the name, or if a path was included, verify that the path is   correct and try again.  At line:1 char:1  + id  + ~~      + CategoryInfo          : ObjectNotFound: (id:String) [], CommandNotFoundException      + FullyQualifiedErrorId : CommandNotFoundException     '
```
There's a tab between `operable` and `program`, I'm not sure if it's same for all Windows (I assume it isn't though), but it seems to be working reliably without last word

```suggestion
              elsif info =~ /command not found|is not recognized as an internal or external command|is not recognized as the name of a cmdlet, function, script file, or operable./
```

Reply:
I updated source branch based on suggestion.